src/cart/crt_rpc.h

```diff
@@ -638,6 +638,10 @@ static inline bool
 crt_rpc_cb_customized(struct crt_context *crt_ctx,
 		      crt_rpc_t *rpc_pub)
 {
+	/* Do not use custom cb in case of SWIM. */
+	if ((rpc_pub->cr_opc>>24)==0xFE)
```

Comment:
(style)  spaces required around that '==' (ctx:VxV)

Reply:
The warning should be fixed but are there not macros and defines to extract these bits?

Reply:
This was particular change was a temporary checkin for Li Wei to try out. Should have mentioned its not for review. This is being discarded and the pull request closed.